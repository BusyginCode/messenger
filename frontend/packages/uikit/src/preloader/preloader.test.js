import React from 'react'
import { shallow } from 'enzyme'
import Preloader from './preloader'

describe('uikit/progress-indicator', () => {
  it('should render correctly', () => {
    expect(shallow(<Preloader />)).toMatchSnapshot()
  })
})
