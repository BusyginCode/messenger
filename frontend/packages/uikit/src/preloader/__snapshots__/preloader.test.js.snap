// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/progress-indicator should render correctly 1`] = `
<Box
  align="center"
  component="div"
  flexGrow={1}
  justify="center"
  padding="s"
>
  <Svg(preloader)
    className="icon"
  />
</Box>
`;
